Foundation/CPAttributedString.j

```diff
@@ -769,6 +771,11 @@
     }];
 }
 
++ (CPAttributedString)attributedStringWithAttachment//(CPTextAttachment)attachment
```

Comment:
can you change this to this method?

+ attributedStringWithImageAttachment:(CPImage)someImage

Reply:
i am aware that this is not cocoa but will get us started.